vmware_aria_operations_integration_sdk/mp_build.py

```diff
@@ -277,30 +370,29 @@ async def build_pak_file(
             )
             manifest = fix_describe(describe_adapter_kind_key, manifest_file)
 
-        # We should ask the user for this before we populate them with default values
-        # Default values are only accessible for authorize members, so we might want to add a message about it
-        docker_registry = get_docker_registry(
-            adapter_kind_key, config_file, docker_registry_arg, **kwargs
-        )
-        tag = manifest["version"] + "_" + str(time.time())
-
-        conf_registry_field, conf_repo_field = get_registry_components(docker_registry)
-
-        # docker daemon seems to have issues when the port is specified: https://github.com/moby/moby/issues/40619
-
-        registry_tag = f"{docker_registry}:{tag}"
-        logger.debug(f"registry tag: {registry_tag}")
-
         try:
             with Spinner("Creating Adapter Image"):
-                build_image(docker_client, path=project_path, tag=f"{registry_tag}")
 
-            with Spinner(f"Pushing Adapter Image to {registry_tag}"):
-                digest = push_image(docker_client, registry_tag)
+                # The first item is the Image object for the image that was built.
+                # The second item is a generator of the build logs as JSON-decoded objects.
+                image, _ = build_image(docker_client, path=project_path)
+
+            # We have to add the Optional type anotation otherwise mypy will mark any line after the while loop as unreachable
+            # https://github.com/python/mypy/issues/5423
```

Comment:
I don't see a while loop around here - did the code get moved without its comment?